TINY-8773: Prevent stripping of "name" and "id" attributes on iframe and img elements (#8011)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

## Unreleased

### Fixed

> - The `name` and `id` attributes of any `img` or `iframe` elements were incorrectly removed #TINY-8773

Reviewer: Don't forget to bump the `modules/tinymce/package.json` version to 5.10.6 as well.

> - The `name` and `id` attributes of any `img` or `iframe` elements were incorrectly removed #TINY-8773

Reviewer: ```suggestion
- The `name` and `id` attributes of some elements were incorrectly removed during serialization #TINY-8773
```
This can happen with other elements, so this just makes it more generic.
